.travis.yml

```diff
@@ -7,28 +13,22 @@ node_js:
   - "6"
   - "4"
 
-sudo: required # Until Yarn repo is added to apt-source-whitelist
-
-addons:
-  apt:
-    packages:
-      - git
+cache:
+  yarn: true
+  directories:
+    - node_modules
 
 branches:
   only:
   - master
   - /^.*-stable$/
 
-cache:
-  # Yarn installed with Travis breaks in Node 4, see https://travis-ci.org/yarnpkg/yarn/jobs/215947941
-  yarn: false
-
 env:
   matrix:
     - TEST_TYPE="build-dist"
     - TEST_TYPE="check-lockfile"
     - TEST_TYPE="lint"
-    - TEST_TYPE="test-ci"
+    - TEST_TYPE="test-only"
```

Comment:
It is still used by Circle CI and Appveyor I think. I'll check that and the Ubuntu setup script in follow-up diffs and remove them if they are unused.